setup.cfg

```diff
@@ -62,9 +62,11 @@ broker =
     # most existing MSAL Python apps do not have the redirect_uri needed by broker.
     #
     # We need pymsalruntime.CallbackData introduced in PyMsalRuntime 0.14
-    pymsalruntime>=0.14,<0.18; python_version>='3.6' and platform_system=='Windows'
+    pymsalruntime>=0.14,<0.19; python_version>='3.6' and platform_system=='Windows'
     # On Mac, PyMsalRuntime 0.17+ is expected to support SSH cert and ROPC
-    pymsalruntime>=0.17,<0.18; python_version>='3.8' and platform_system=='Darwin'
+    pymsalruntime>=0.17,<0.19; python_version>='3.8' and platform_system=='Darwin'
+    # PyMsalRuntime 0.18+ is expected to support broker on Linux
+    pymsalruntime>=0.18,<0.19; python_version>='3.8' and platform_system=='Linux'
```

Comment:
Have you released or do you plan to release 0.18.0 for Linux? I saw right now it's still 0.17.1

Reply:
I have not yet released 0.18.0, but i'll be making that release soon, and this will be the first package with linux broker support

Reply:
**suggestion:** This PR shall be changed to a draft. We won't merge it in until pymsalruntime 0.18 being released.

Reply:
Can we ensure the next release of `pymsalruntime` have whls built for Python 3.13 as well? The latest version of `pymsalruntime` doesn't seem to have that.